src/components/MovieCard/models.ts

```diff
@@ -0,0 +1,7 @@
+export interface MovieCardProps {
+    posterUrl: string,
+    link: string,
+    title: string,
+    genre: string,
```

Comment:
Range of values for `genre` property is quite limited, so, for type safety, you may constrain those, using `enum`:
```ts
export enum Genre {
   ACTION = 'action',
   COMEDY = 'comedy'
   ..
}

export interface Movie {
   ..
   genre: Genre,
   ..
}
```